4.8/architectures

```diff
@@ -0,0 +1,3 @@
+bashbrew-arch   variants
```

Comment:
yes! it's local specialization :-) I did that. I felt we may need it someday. Thanks for the review!